fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2126,6 +2126,30 @@ public class Config extends ConfigBase {
     @ConfField
     public static long binlog_max_size = Long.MAX_VALUE; // no limit
 
+    /**
+     * Enable check if the cluster is under safe mode or not
+     **/
+    @ConfField(mutable = true)
+    public static boolean enable_safe_mode_check = true;
+
+    /**
+     * The safe mode checker thread work interval
+     */
+    @ConfField(mutable = true)
+    public static long safe_mode_checker_interval_sec = 5;
```

Comment:
Do we need the other three configs?  
How about we can choose `min(0.9 * disk_capacity, 50GB)` to determine whether it will abort the ingestion.